examples/metrics/Dockerfile-zxp

```diff
@@ -23,9 +25,11 @@ RUN echo '{\n\
 # ---
 # Stage 2: Final image with nothing but binary and default config file
 # ---
-FROM scratch AS final
-COPY --from=builder /go/src/github.com/project-zot/zot/bin/zxp /zxp
+FROM gcr.io/distroless/base AS final
+ARG OS
+ARG ARCH
+COPY --from=builder /go/src/github.com/project-zot/zot/bin/zxp-$OS-$ARCH /usr/bin/zxp
```

Comment:
Better to use zxp-linux-amd64 directly and get rid of OS & ARCH args

Reply:
No why? I can reuse the same Dockerfile and it gets OS and ARCH from ci/cd

Reply:
The idea was that kind-setup.sh script was never tested on other platforms to prove it works.
If the idea was to bring a kind cluster with all the zot pods installed (+ prometheus) than we need to validate that script works on different flavours of Mac processors too.